src/components/form-elements/_select.scss

```diff
@@ -23,7 +23,7 @@ $includeHtml: false !default;
     border: $selectBorderWidth solid $selectBackground;
 
     &__element {
-      @include uppercaseText;
+      @include uppercaseText(0);
```

Comment:
This removes `letter-spacing` if you not specify this value letter spacing is `0.03rem` by default